docs/features/private-wlan.rst

```diff
@@ -4,6 +4,14 @@ Private WLAN
 It is possible to set up a private WLAN that bridges the WAN port and is separated from the mesh network.
 Please note that you should not enable ``mesh_on_wan`` simultaneously.
 
+The private WLAN is encrypted via WPA2 by default. On devices with enough flash and a supported radio,
+WPA3 or WPA2/WPA3 mixed-mode can be used instead of WPA2. For this to work, the ``wireless-encryption-wpa3``
+feature has to be added to the list of your supported features.
+
+Additionally, IEEE 802.11w management frame protection should be enforced for WPA2/WPA3, however this
+can lead to connectivity problems for older clients. In this case, management frame protection can be
+set to optional or disabled completely using advanced settings. 
```

Comment:
```suggestion
downgraded to be either optional or completely disabled using advanced settings. 
```

Reply:
Altered this part according to @NeoRaider s previous feedback (which was overwritten by accident).